lib/src/library.dart

```diff
@@ -74,18 +74,60 @@ class LibraryReader {
   Iterable<ClassElement> get classElements =>
       element.definingCompilationUnit.types;
 
-  Iterable<Element> _getElements(CompilationUnitMember member) {
+  static Iterable<Element> _getElements(CompilationUnitMember member) {
     if (member is TopLevelVariableDeclaration) {
       return member.variables.variables
           .map(resolutionMap.elementDeclaredByVariableDeclaration);
     }
     var element = resolutionMap.elementDeclaredByDeclaration(member);
-
     if (element == null) {
-      print([member, member.runtimeType, member.element]);
-      throw new Exception('Could not find any elements for the provided unit.');
+      throw new StateError(
+          'Could not find any elements for the provided unit.');
     }
-
     return [element];
   }
+
+  /// Returns the identifier prefix of [element] if it is referenced by one.
+  ///
+  /// For example in the following file:
+  /// ```
+  /// import 'bar.dart' as bar;
+  ///
+  /// const b = const bar.Bar();
+  /// ```
+  ///
+  /// ... we'd assume that `b`'s element has a prefix of `bar`.
```

Comment:
I'm still confused by this. There isn't anywhere we'd have the code `bar.b` - so why is it that `b` is considered to have a prefix?

Reply:
Yeah this isn't an accurate example anymore. I'll update it.

Reply:
Done.